astropy/visualization/wcsaxes/formatter_locator.py

```diff
@@ -90,6 +91,16 @@ def minor_locator(self, spacing, frequency, value_min, value_max):
         values = np.delete(values, np.s_[index::frequency])
         return values * minor_spacing * self._unit
 
+    @property
+    def format_unit(self):
+        return self._format_unit
+
+    @format_unit.setter
+    def format_unit(self, unit):
```

Comment:
Could the setter also take a string like 'hourangle' or 'deg'?

Reply:
Yep, I've updated this and ``set_format_unit`` to accept anything that can initialize ``Unit``

Reply:
Looks good to me!